v21.2/use-cloud-storage-for-bulk-operations.md

```diff
@@ -225,6 +225,87 @@ A custom root CA can be appended to the system's default CAs by setting the `clo
 
 </section>
 
+## Storage permissions
+
+This section describes the minimum permissions required to run CockroachDB bulk operations. While we outline information on the necessary permissions to access Amazon S3 and Google Cloud Storage when running these operations, the provider's documentation provides detail on the setup process and different options regarding access management.
+
+Depending on the actions a bulk operation performs, it will require different access permissions to a cloud storage bucket. 
+
+This table outlines the actions that each operation performs against the storage bucket:
+
+Operation                   | Permission                   | Description                                                                     
+----------------------------+------------------------------+---------------------------------------------------
+[Backup](backup.html)       | Write<br>Get<br>List<br>Delete | Write/Get: Backups write the backup data to the storage bucket. During a backup job, a `BACKUP CHECKPOINT` file will be written that tracks the progress of the backup. Should the backup resume after a [pause](pause-job.html), it will read the checkpoint file to find a suitable resume point. This can include overwriting backup work that had happened before the pause.<br> List: Backups need list access to the files already in the bucket. If the backup is paused, or there is a retryable error, the backup will read the `BACKUP CHECKPOINT` file.<br> Delete: Backups may delete files and progress during the job after a pause in order to resume the job from a certain point.
+[Restore](restore.html)     | List<br><br>Get                    | List: During a restore job, it is necessary to pass the backup's directory. This contains a manifest file that describes the backup's metadata and data. Restores require access to read these files from the storage bucket to process the backup correctly. <br> Get: Restores need access to retrieve files from the backup.
```

Comment:
For List, I'm not sure this describes why we need the permission since it only mentions that the directory contains a manifest and that we need read on that manifest. Do we want these to be comprehensive lists of when we use the operations?  The number of uses of List grew between 21.2 and 22.1, but we can easily generate the list for both. if that is the goal.